static/app/components/smartSearchBar/actions.tsx

```diff
@@ -0,0 +1,209 @@
+import {browserHistory, withRouter, WithRouterProps} from 'react-router';
+import styled from '@emotion/styled';
+
+import {openModal} from 'app/actionCreators/modal';
+import {pinSearch, unpinSearch} from 'app/actionCreators/savedSearches';
+import Access from 'app/components/acl/access';
+import Button from 'app/components/button';
+import MenuItem from 'app/components/menuItem';
+import {IconAdd, IconPin, IconSliders} from 'app/icons';
+import {t} from 'app/locale';
+import {SavedSearch} from 'app/types';
+import {trackAnalyticsEvent} from 'app/utils/analytics';
+import CreateSavedSearchModal from 'app/views/issueList/createSavedSearchModal';
+
+import SmartSearchBar from './index';
```

Comment:
Is this going to cause some circular import issues?

Reply:
It does not.